nanoFramework.System.IO.FileSystem.nuspec

```diff
@@ -21,6 +21,7 @@ This package requires a target with System.IO.FileSystem v$nativeVersion$ (check
     <dependencies>
       <dependency id="nanoFramework.CoreLibrary" version="1.14.2" />
       <dependency id="nanoFramework.Runtime.Events" version="1.11.6" />
+      <dependency id="nanoFramework.System.Text" version="1.2.37" />
```

Comment:
This should not be declared as a dependency because it's already a dependency of nanoFramework.System.IO.Streams